packages/compass-e2e-tests/tests/my-queries-tab.test.ts

```diff
@@ -267,121 +268,147 @@ describe('Instance my queries tab', function () {
     const namespace = await browser.getActiveTabNamespace();
     expect(namespace).to.equal('test.numbers');
   });
+});
+
+describe('when a user has a saved query associated with a collection that does not exist', function () {
+  let compass: Compass;
+  let browser: CompassBrowser;
+  const connectionName = connectionNameFromString(DEFAULT_CONNECTION_STRING);
+  const favoriteQueryName = 'list of numbers greater than 10 - query';
+  const newCollectionName = 'numbers-renamed';
+
+  before(async function () {
+    skipForWeb(this, 'saved queries not yet available in compass-web');
+
+    compass = await init(this.test?.fullTitle());
+    browser = compass.browser;
+  });
```

Comment:
we probably moved it out of root describe block and that's why we need to skipForWeb here as well, should we just move  it back in, unless this was intentional?
If it was I recall seeing in compass somewhere a rule about only having one suite per test file.

Reply:
Like I said in the description I moved it deliberately.

Reply:
Actually there's no real reason for that second beforeEach to exist and be any different. Fixing.